TypeCobol/Compiler/CodeModel/SymbolTable.cs

```diff
@@ -783,6 +783,14 @@ public IList<Section> GetSection(string name)
             return EmptySectionList;
         }
 
+        /// <summary>
+        /// Get all sections in the current scope.
+        /// </summary>
+        /// <returns>The collection of section names</returns>
+        public IEnumerable<Section> GetSections(Func<Section, bool> predicate)
+        {
+            return Sections.Values.SelectMany(s => s).Where(predicate).Distinct();
```

Comment:
Following PR #2560, HashSet is faster than using `Distinct`.
But in this case, the `SymbolTable.Sections` contains all ocurrences of sections only once.
So `Distinct` is not useful.

In fact it can also be removed in `GetParagraphs(Func<Paragraph, bool> predicate)`